### ISSUE TITLE
Export current schemes does not include all color information

### ISSUE THREAD

**Issue by LokiMidgard** [Microsoft/console#392]
When exporting a scheme and reaplieing it after another scheme was applied, it will not result in the original colors.

* Your Windows build number: Microsoft Windows [Version 10.0.17763.316]

* What you're doing and what's happening:
```
ColorTool.exe -o .\Documents\WindowsPowerShell\programs\colortool\schemes\powershell-default.ini
ColorTool.exe onehalfdark
ColorTool.exe powershell-default

```

* What's wrong / what should be happening instead:
oneHalfDark is changing the background color. when reaplieing the previous exported scheme the background color index is not restored.

**Comment by LokiMidgard** [comment]
I think I could look into it and create a pull request.

Is there a specific documentation to the used format of the ini file? Or can I just add more sections?

**Comment by LokiMidgard** [comment]
I created a [pullrequest](#400 ) that adds support in persisting the color indexes.

**Comment by LokiMidgard** [comment]
should be resolved by #400

**Comment by miniksa** [comment]
Thank you!
